src/Components/WebAssembly/Samples/HostedBlazorWebassemblyApp/Client/HostedBlazorWebassemblyApp.Client.csproj

```diff
@@ -15,4 +15,10 @@
     <ProjectReference Include="..\Shared\HostedBlazorWebassemblyApp.Shared.csproj" />
   </ItemGroup>
 
+  <Target Name="_SkipXmlDocFiles" BeforeTargets="_ResolveBlazorWasmOutputs">
```

Comment:
I recall you mentioned something earlier about this but I don't recall the details. Does anything need to change in project templates to have this workaround for Blazor WebAssembly?

Reply:
This is just because our build is "special" and ends up putting extra stuff on the itemgroups we normally look at. The workarounds here don't really apply to "regular" builds a customer would do.

That said, all those things are addressed [here]( https://github.com/dotnet/sdk/pull/19080), so this commit is only until we update to an sdk containing the fixes, at which point, I'll revert the commit with these fixes